Automattic/jetpack
# Boost: Update getting started flow

Fixes Automattic/boost-cloud#182

## Proposed changes:
- [x] Connection screen is removed, only getting started page is shown
- [x] Site connection is done automatically when user chooses a plan from getting started page
- [x] If a site connection already exists, you still should see a getting-started page, but choosing a plan shouldn't reinitiate a connection
- [x] If there is any error while establishing site connection, show a snackbar
- [x] Fix e2e tests to work with the updates (currently failing at trying to [establish connection via CLI](https://github.com/Automattic/jetpack/actions/runs/4192667237/jobs/7270837523#step:11:87))

### Other information:


## Jetpack product discussion


None

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* You should see the getting started page with pricing table
* Make sure the proposed changes are working
* Try combinations of: free plan, paid plan, jetpack already installed and connected, without preexisting connection)
